stable/datadog/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: datadog
-version: 2.2.10
```

Comment:
As per the image version requirement, it should be a 2.3 as it'd break on upgrade if users specified an image tag.

Reply:
Version bumped.